include/libdnf/base/goal.hpp

```diff
@@ -54,53 +54,285 @@ class Goal {
     explicit Goal(const libdnf::BaseWeakPtr & base);
     explicit Goal(libdnf::Base & base);
     ~Goal();
+
+    // TODO(jmracek) Implemented not yet
```

Comment:
(FWIW) "not yet implemented" (unless you're going Yoda style :wink: )

Reply:
Fixed